media/css/base/banners/daylight-promo.scss

```
@@ -0,0 +1,130 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+$font-path: '/media/fonts';
+$image-path: '/media/protocol/img';
+
+@import '../../../protocol/css/includes/lib';
+
+#firefox-daylight-promo {
+    &.c-banner {
+        @include clearfix;
+        background: $color-orange-70 linear-gradient(to right, #fca134, #e1294f);
+        border-bottom: 1px solid rgba(0, 0, 0, .25) inset;
+        color: $color-ink-80;
+        position: relative;
+        z-index: 3;
+
+        // hide by default if JS is available to avoid flicker
+        // (if visitor previously dismissed)
+        .js & {
+            display: none;
+        }
+
+        // conditional class used to display the banner.
+        &.c-banner-is-visible {
+            display: block;
+        }
+    }
+
+    .c-banner-title {
+        @include bidi(((padding-right, $spacing-lg, 0), (padding-left, 0, $spacing-lg),)); // make room for the close button
+        @include text-title-xs;
+    }
+
+    @media #{$mq-md} {
+        .c-banner-main {
+            display: flex;
+            align-items: center;
+        }
+
+        .c-banner-title {
+            margin: 0;
+            padding: 0;
+        }
+
+        .c-cta {
+            padding: 0 $spacing-xl;
+            line-height: 0;
+        }
+    }
+
+
+    // Conditional content
+    .hidden-on-desktop {
+        display: none;
+    }
+
+    .android,
+    .ios {
```

It looks like the conditional classes aren't quite working here:
```suggestion
    .android &,
    .ios & {
```